src/main/resources/instructorEmailTemplate-newAccountRequestAcknowledgement.html

```diff
@@ -0,0 +1,65 @@
+<p>Hello, ${name}</p>
+
+<p>
+  Thank you for submitting an account request. This is what you have submitted:
+</p>
+
+<div>
+  <table style="max-width:600px;border:1px solid black;">
+    <tr>
+      <td style="padding:5px;">
+        <strong>
+          Full Name
+        </strong>
+      </td>
+      <td style="padding:5px;">
+        ${name}
+      </td>
+    </tr>
+
+    <tr>
+      <td style="padding:5px;">
+        <strong>
+          Country & Institute
+        </strong>
+      </td>
+      <td style="padding:5px;">
+        ${institute}
+      </td>
+    </tr>
+
+    <tr>
+      <td style="padding:5px;">
+        <strong>
+          Email Address
+        </strong>
+      </td>
+      <td style="padding:5px;">
+        ${emailAddress}
+      </td>
+    </tr>
+
+    <tr>
+      <td style="padding:5px;">
+        <strong>
+          Home Page URL<br>& Comments
+        </strong>
+      </td>
+      <td style="padding:5px;">
+        ${comments}
+      </td>
+    </tr>
+  </table>
+</div>
+
+<p>
+  Your request will be reviewed within 24 hours. We will send another email once your request has been accepted.
+</p>
+<p>
+  If you have any additional queries, please feel free to contact us at ${supportEmail}. 
```

Comment:
```suggestion
  If you have any additional queries, please feel free to contact us at ${supportEmail}.
```

I believe you may have left an extra space here. It's causing the test to fail. Notice the difference in length in the following when highlighted.

![image](https://github.com/TEAMMATES/teammates/assets/65202977/650ba318-3842-444d-83bb-93cf37d5e616)

Reply:
Thanks for spotting that! I'll fix the test cases